src/components/overview.jsx

```diff
@@ -45,19 +45,35 @@ class Overview extends React.Component {
       }
     }
   }
+  renderMarkdown() {
+    const { customRenderers, markdown } = this.props;
+    let renderedMarkdown = marked(markdown);
+
+    if (customRenderers) {
+      const renderer = new marked.Renderer();
+      Object.keys(customRenderers).forEach((key) => renderer[key] = customRenderers[key]);
```

Comment:
Since you're copying object properties, you could do `Object.assign(renderer, customRenderers)` instead of the iteration.